tests/system/apps/faultdomain-base-app.json

```diff
@@ -0,0 +1,8 @@
+{
+  "id": "/fault-domain",
```

Comment:
Hm, this is not much different than other app definitions. Should we just use the sleep app and change the id?

Reply:
Yes, it's very close to the sleep app. I created a new app definition purely for separation of concerns.